backend/qemu.pm

```diff
@@ -610,6 +610,8 @@ sub start_qemu ($self) {
                 }
             }
             $qemubin = find_bin('/usr/bin/', @execs) unless $qemubin;
+            # In Enterprise Linux QEMU binary located at /usr/libexec/qemu-kvm
+            $qemubin = find_bin('/usr/libexec/', 'qemu-kvm') unless $qemubin;
```

Comment:
Actually this can be simplified to just

```suggestion
            $qemubin = find_bin('/usr/bin/', @execs) // find_bin('/usr/libexec/', @execs) unless $qemubin;
```

Reply:
Okay, let me test it.

Reply:
Tested, works. Updated the patch.